pkg/client/auth_providers.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2017 the Velero contributors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package client
+
+// Make sure we import the client-go auth provider plugins.
+
+import (
+	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
```

Comment:
According to the Revive testing data. It's better to move the comment line right above the import lines.

https://github.com/mgechev/revive/pull/501/files

```suggestion
package client

import (
	// Make sure we import the client-go auth provider plugins.
	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
		_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
)
```

Reply:
@blackpiglet, I moved the comment. Thank you for pointing that out!